electron/build/template-package.json

```diff
@@ -26,7 +26,7 @@
     "patch": "ncp ./patch/backend/main.js ./src-gen/backend/main.js"
   },
   "engines": {
-    "node": ">=14.0.0 <15"
+    "node": ">=15.0.0 <17"
```

Comment:
@kittaakos wouldn't it be better if we enforce using node 16 here, so that if anyone who accidentally tries to build the IDE with node 15 on an M1 would be blocked here?

Reply:
I do not mind, but it should work with 15+ too. Tell me the engine constraints, and I do the update. Thanks!

Reply:
I was wondering what would happen if someone with an M1 tries to build with node 15. I think they should be warned with a meaningful error that suggests to use Node 16, and enforcing it could be a solution. But maybe they would get a meaningful error anyway (I need to try it).

Actually I don't have a strong opinion on this and trust yours more since you worked on it. What do you think?

Reply:
Node 15 is required to pick up the `arm64` arch on M1 instead of `x64`., but looking into the Node [release schedule](https://github.com/nodejs/release#release-schedule), you have convinced me to stick to `>=16`. Thanks for looking into the changeset 👍

Reply:
I'm glad I helped you taking the decision, even if I suggested it for the wrong reason 🤣